packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -29,9 +36,32 @@ import {
   getAttributeByOriginAndCodeErrorMapper,
   getAttributesByNameErrorMapper,
 } from "../utilities/errorMappers.js";
+import { readModelServiceBuilderSQL } from "../services/readModelServiceSQL.js";
+
+const db = makeDrizzleConnection(config);
+const attributeReadModelServiceSQL = attributeReadModelServiceBuilderSQL(db);
+const tenantReadModelServiceSQL = tenantReadModelServiceBuilderSQL(db);
 
 const readModelRepository = ReadModelRepository.init(config);
-const readModelService = readModelServiceBuilder(readModelRepository);
+const oldReadModelService = readModelServiceBuilder(readModelRepository);
+const readModelServiceSQL = readModelServiceBuilderSQL(
+  db,
+  attributeReadModelServiceSQL,
+  tenantReadModelServiceSQL
+);
+
+const SQLFeatureFlagEnabled =
+  FeatureFlagSQL.safeParse(process.env).success && config.featureFlagSQL;
+
+const SQLReadModelConfigured = ReadModelSQLDbConfig.safeParse(
+  process.env
+).success;
```

Comment:
Same here, since parsing was already done you could just use the values in config

```suggestion
const SQLReadModelConfigured = config.readmodelSqlDbHost && config.readmodelSqlDbPort 
```

Reply:
Updated